stable_baselines3/common/base_class.py

```diff
@@ -545,15 +545,18 @@ def predict(
         state: Optional[np.ndarray] = None,
         mask: Optional[np.ndarray] = None,
         deterministic: bool = False,
-    ) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+    ) -> Tuple[np.ndarray, Optional[Tuple[np.ndarray, ...]]]:
         """
-        Get the model's action(s) from an observation
+        Get the policy action and state from an observation (and optional state).
```

Comment:
What is the "optional state"? I assume you are referring to things like LSTM where it returns two ndarrays for a hidden state but something else might just have one ndarray? Maybe just easier to say

`Get the policy action and an optional hidden state`